pkg/data/mysql.go

```diff
@@ -15,3 +15,430 @@ limitations under the License.
+			if err != nil {
+				log.Fatalln(err)
+			}
+
+			tunnel, err := lib.TunnelToDBService(opts.config, dbName, namespace, api.MySQLDatabasePort)
+			if err != nil {
+				log.Fatal("couldn't create tunnel, error: ", err)
+			}
+
+			err = opts.verifyDataExecCmd(tunnel, rows)
+			if err != nil {
+				log.Fatal(err)
+			}
+
+			tunnel.Close()
+		},
+	}
+
+	myVerifyCmd.Flags().IntVarP(&rows, "rows", "r", 10, "rows in ")
+
+	return myVerifyCmd
+}
+
+func (opts *mysqlOpts) verifyDataExecCmd(tunnel *portforward.Tunnel, rows int) error {
+	if rows <= 0 {
+		return fmt.Errorf("rows need to be greater than 0")
+	}
+	var command string
+
+	command = ` 
+		USE mysql;
+		SELECT COUNT(*) FROM kubedb_table; 
+	`
+
+	out, err := opts.executeCommand(tunnel.Local, command)
+	if err != nil {
+		return err
+	}
+
+	var output []string
+	output = strings.Split(out, "\n")
+
+	totalKeys, err := strconv.Atoi(strings.TrimPrefix(output[1], " "))
+	if err != nil {
+		return err
+	}
+	if totalKeys >= rows {
+		fmt.Printf("\nSuccess! MySQL database %s/%s contains: %d keys\n", opts.db.Namespace, opts.db.Name, totalKeys)
+	} else {
+		fmt.Printf("\nError! Expected keys: %d . MySQL database %s/%s contains: %d keys\n", rows, opts.db.Namespace, opts.db.Name, totalKeys)
+	}
+	return nil
+}
+
+func DropMySQLDataCMD(f cmdutil.Factory) *cobra.Command {
+	var (
+		dbName      string
+		mysqlDBName string
+	)
+
+	myDropCmd := &cobra.Command{
+		Use: "mysql",
+		Aliases: []string{
+			"my",
+			"mysql",
+		},
+		Short:   "Verify rows in a MySQL database",
+		Long:    `Use this cmd to verify data in a mysql object`,
+		Example: `kubectl dba drop mysql -n demo sample-mysql`,
+		Run: func(cmd *cobra.Command, args []string) {
+			if len(args) == 0 {
+				log.Fatal("Enter mysql object's name as an argument.")
+			}
+			dbName = args[0]
+
+			namespace, _, err := f.ToRawKubeConfigLoader().Namespace()
+			if err != nil {
+				klog.Error(err, "failed to get current namespace")
+			}
+
+			opts, err := newMySQLOpts(f, dbName, namespace, mysqlDBName)
+			if err != nil {
+				log.Fatalln(err)
+			}
+
+			tunnel, err := lib.TunnelToDBService(opts.config, dbName, namespace, api.MySQLDatabasePort)
+			if err != nil {
+				log.Fatal("couldn't create tunnel, error: ", err)
+			}
+
+			err = opts.dropDataExecCmd(tunnel)
+			if err != nil {
+				log.Fatal(err)
+			}
+
+			tunnel.Close()
+		},
+	}
+
+	return myDropCmd
+}
+
+func (opts *mysqlOpts) dropDataExecCmd(tunnel *portforward.Tunnel) error {
+	var command string
+	command = ` 
+		USE mysql;
+		DROP TABLE IF EXISTS kubedb_table;
+	`
+	_, err := opts.executeCommand(tunnel.Local, command)
+	if err != nil {
+		return err
+	}
+	fmt.Printf("\nSuccess: All the CLI inserted rows DELETED from MySQL database %s/%s.\n", opts.db.Namespace, opts.db.Name)
+
+	return nil
+}
+
+type mysqlOpts struct {
+	db          *api.MySQL
+	dbImage     string
+	config      *rest.Config
+	client      *kubernetes.Clientset
+	mysqlDBName string
```

Comment:
why `musqlDBName` variable needed ?

Reply:
also, run `make lint` & fix the linter errors

Reply:
At first, we thought that might be user can add any database but my current implementation is based on `mysql` database only and kubedb_table where the user can add data. That's why add the `mysqlDBName` variable.

Reply:
ok, remove then